dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE9Action.java

```diff
@@ -133,6 +135,11 @@ public JakartaEE9Action addFeatures(Set<String> addFeatures) {
         return (JakartaEE9Action) super.addFeatures(addFeatures);
     }
 
+    public JakartaEE9Action addAdditionalFeatures(Set<String> additionalFeatures) {
```

Comment:
If we are going to add this new API I think it makes sense to add it in `FeatureReplacementAction` so it can apply to all subclasses, rather than only putting it on the `JakartaEE9Action`

Reply:
Could we do a different name for this method? Without knowing the background it's not clear what "additional features" mean. The purpose of this new method is to "add a feature that will always be added to configs" (also referred to as "force adding"). So perhaps name change `addAdditionalFeatures` --> `forceAddFeature` or `alwaysAddFeature`. Another option could be overloading `addFeatures` with a boolean param for force add like: `addFeatures(Set<String>, boolean forciblyAdd)`

Whatever method signature we go with, javadoc please!